documentation/common/snip-create-kubernetes-project.adoc

```diff
@@ -2,12 +2,12 @@
 +
 [options="nowrap",subs="attributes"]
 ----
-kubectl create namespace enmasse
+kubectl create namespace {ProductNamespace}
 ----
 
-. Set the `enmasse` namespace as the default namespace:
+. Set the `{ProductNamespace}` namespace as the default namespace:
 +
 [options="nowrap",subs="attributes"]
 ----
-kubectl config set-context $(kubectl config current-context) --namespace=enmasse
+kubectl config set-context $(kubectl config current-context) --namespace={ProductNamespace}
 ----
```

Comment:
Add:
"where <{ProductNamespace}> is the namespace where {ProductName} is deployed."